cms/static/cms/js/modules/cms.page_type_select.js

```diff
@@ -0,0 +1,26 @@
+//##############################################################################
```

Comment:
I would agree with you if this was something that all wizards used, but it is not. This is JS for a wizard-specific field and if included for all wizards could affect 3rd party wizards. I have, however, pulled out the inline JS and CSS that you contributed from the base template into their own files.